main.py

```diff
@@ -3,9 +3,19 @@
 from datetime import date
 from datetime import datetime
 
+class Record:
+    def __init__(vehicle, date, mileage, litres, cost, currency):
+        self.vehicle = vehicle
+        self.date = date
+        self.mileage = mileage
+        self.litres = litres
+        self.cost = cost
+        currency = currency
```

Comment:
Is it supposed to be self.currency not just currency?

Reply:
yep, you got it! I'll fix that up and then it should be good to merge